website/docs/developer-section/quickstart/create-verifiable-document-issuer.md

```diff
@@ -40,13 +40,13 @@ If not, follow the steps below:
 
 ### Verifiable document components
 
-Before starting on this code tutorial, it will be helpful to understand the components involved in the creation, issuance and verification of a verifiable document.
+Before starting on this code tutorial, it will be helpful to understand the components involved in the creation, issuance, and verification of a verifiable document.
 
 You can read more about the components [here](/docs/integrator-section/verifiable-document/ethereum/document-store-overview).
 
 ## Overview
 
-You will build a single-page application, which helps the user perform the following:
+You will build a single-page application, so that the user can perform the following:
```

Comment:
```suggestion
You will build a single-page application that performs the following
```

Reading through this page we changed a lot of things to "so that" which feels strange and repetitive, can we add some variations? not sure if the tense is in line with the rest

Reply:
updated to:

> You will build a single-page application with these features:
>
> 1. The users can connect to their MetaMask wallet.
> 2. They can deploy their own document store.
> ...